CONTRIBUTING.md

```diff
@@ -96,7 +96,7 @@ git clone https://github.com/<your-username>/Scribe-Data.git
 # Navigate to the newly cloned directory.
 cd Scribe-Data
 # Assign the original repo to a remote called "upstream".
-git remote add upstream https://github.com/scribe-org/Scibe-Data.git
+git remote add upstream https://github.com/scribe-org/Scribe-Data.git
```

Comment:
Thanks @ikeadeoyin! Would you also do the same in the `README.md`? Looks like there's an equivalent typo there as well.

Reply:
Hello @wkyoshida, I have made changes to the `README.md` as well.